### DISCUSSION TITLE
Xaml style

### DISCUSSION THREAD

**Discussion by Vian1986** [Vian1986/Tafe22S108#4] [General]
hi @fionaperr318 , I use below style for my Xaml design:
	<Page.Resources>
		<Style TargetType="TextBlock">
			<Setter Property="Width" Value="250"/>
		</Style>
		<Style TargetType="TextBox">
			<Setter Property="Width" Value="250"/>
		</Style>
		<Style TargetType="StackPanel">
			<Setter Property="Margin" Value="10"/>
		</Style>
	</Page.Resources>
Do you think we should use the same style?

**Comment by fionaperr318** [comment]
Thanks for your question, I agree that we should use same style so that our interface would be neat and consistent.
